controllers/profiles/common/properties/managed.go

```diff
@@ -183,10 +185,10 @@ func NewManagedPropertyHandler(workflow *operatorapi.SonataFlow, platform *opera
 	return handler.withKogitoServiceUrl(), nil
 }
 
-// ApplicationManagedProperties immutable default application properties that can be used with any workflow based on Quarkus.
-// Alias for NewManagedPropertyHandler(workflow).Build()
-func ApplicationManagedProperties(workflow *operatorapi.SonataFlow, platform *operatorapi.SonataFlowPlatform) (string, error) {
-	p, err := NewManagedPropertyHandler(workflow, platform)
+// ImmutableApplicationProperties immutable default application properties that can be used with any workflow based on Quarkus.
+// Alias for NewAppPropertyHandler(workflow).Build()
+func ImmutableApplicationProperties(ctx context.Context, c client.Client, workflow *operatorapi.SonataFlow, platform *operatorapi.SonataFlowPlatform) (string, error) {
```

Comment:
Why have you changed this function name?

Reply:
Caused by the code rebase. Corrected